clients/java/src/main/java/io/camunda/zeebe/client/impl/oauth/OAuthCredentialsCache.java

```diff
@@ -110,11 +110,11 @@ public synchronized ZeebeClientCredentials computeIfMissingOrInvalid(
         readCache()
             .get(endpoint)
             .flatMap(
-                zeebeClientCredentials -> {
-                  if (!zeebeClientCredentials.isValid()) {
+                camundaClientCredentials -> {
+                  if (!camundaClientCredentials.isValid()) {
```

Comment:
🔧 Should this stay as is? The name change doesn't make sense for the Zeebe Client, I guess?

Reply:
I didn't change vars names, I think it can be a follow-up issue